project/iterative_sorting.py

```diff
@@ -6,29 +6,77 @@ def selection_sort( arr ):
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-             
+        for j in range(cur_index, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
 
 
 
         # TO-DO: swap
-
+        temp = arr[smallest_index]
```

Comment:
nice! this kind of swap also works in python
a, b = b, a
or
arr[i], arr[i+1] = arr[i+1], arr[i]